File: Desafios de codigo/Javascript/ServiceCheck.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Digite o nome do servico: ', (serviceName) => {
    rl.question('Digite os dados do cliente: ', (clientData) => {
        let clientPieces = clientData.trim().split(',');
        let found = false;

        for (let i = 1; i < clientPieces.length; i++) {
            if (clientPieces[i].trim() === serviceName) found = true;
        }

        if (found) console.log('Servico encontrado');
        else console.log('Servico nao encontrado');

        rl.close();
    });
});

File: Desafios de codigo/C++/ServiceCheck.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    std::string serviceName;
    std::string clientData;
    std::string clientPieces[100];
    bool found = false;

    std::cout << "Digite o nome do servico: ";
    std::cin >> serviceName;

    std::cin.ignore();

    std::cout << "Digite os dados do cliente: ";
    std::getline(std::cin, clientData);

    int j = 0;
    int k = 0;
    for (int i = 0; i < clientData.length(); i++) {
        if (clientData[i] == ',' || i == clientData.length() - 1) {
            clientPieces[j] = clientData.substr(k, i - k);
            k = i + 1;
            j++;
        }
    }

    for (int i = 1; i < j; i++) {
        if (clientPieces[i] == serviceName) {
            found = true;
            break;
        }
    }

    if (found) std::cout << "Servico encontrado!" << std::endl;
    else std::cout << "Servico nao encontrado!" << std::endl;

}